components/automate-ui/src/app/page-components/profile/profile.component.html

```diff
@@ -45,7 +45,7 @@
               #focusElement>About</button>
           </li>
           <li>
-            <a href="https://www.chef.io/online-master-agreement/" target="_blank"
+            <a href="https://www.chef.io/online-master-agreement/" target="_blank" rel="noopener"
```

Comment:
🎉 Nice to bring in `noopener` for added security--I was not aware of that at all.
Here's a link for posterity explaining it: https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/